core/controller/src/main/scala/whisk/core/controller/Meta.scala

```diff
@@ -310,6 +313,11 @@ trait WhiskMetaApi
         Identity.get(authStore, namespace)
     }
 
+    val validateWhiskRaw: Directive1[String] = {
+        val value = headerValueByName("x-ow-raw")
```

Comment:
I think @rabbah proposed the header name is `X-OW-RAW-HTTP` instead
Also don't repeat your self, create a val with the key and don't hard code the values in multiple places, allow to update in one single place.